rpi_backlight/__init__.py

```diff
@@ -160,13 +191,26 @@ def power(self) -> bool:
         :setter: Set the display power on or off.
         :type: bool
         """
-        # 0 is on, 1 is off
-        return not self._get_value("bl_power")
+
+        if self._board_type == BoardType.RASPBERRY_PI:
+            # 0 is on, 1 is off
+            return not self._get_value("bl_power")
+        elif self._board_type == BoardType.TINKER_BOARD:
+            if get_brightness_value() == 0:
+                value = 255
+            else:
+                value = 0
+            self._set_value("tinker_mcu_bl", value)
```

Comment:
- `get_brightness_value` is not defined
- This function should not call `_set_value` but return a boolean (this is the getter function)

Reply:
Sorry about that, that was residual code from my version 1. 
I am not able to test this until tonight, so coding a bit blind.
Will test it all tonight (dont accept the pull yet)

Reply:
no worries!